src/source/Ice/IceAgent.h

```diff
@@ -200,6 +200,9 @@ struct __IceAgent {
 
     // store transaction ids for stun binding request.
     PTransactionIdStore pStunBindingRequestTransactionIdStore;
+
+    // Add ICE related Stats
+    PIceStats pIceStats;
```

Comment:
Don't include these. We will be adding them later - not sure if we will keep the public struct or an internal one that can carry more info needed for tracking. Also, you are including a pointer here.

Reply:
Done